### ISSUE TITLE
Impossible to delete selected label [NER sequence labelling]

### ISSUE THREAD

**Issue by Joanna-Wojdylo** [doccano/doccano#1571]
Hi Team,

Something happened, I believe in versions above 1.4.1, and now I cannot work properly with sequence labelling, please see below print screens from version 1.4.1 and 1.5.2, **the new design won't allow me to remove selected labels**. I tried both Docker and pip installation of 1.5.2 and it was the same (I tested 1.4.1 using pip installation and it's working perfectly fine). Is there a chance for any quick fix/advice? 

From version 1.4.1: You can delete each label by clicking x
![Zrzut ekranu 2021-11-22 o 12 45 12](https://user-images.githubusercontent.com/62895686/142864336-1d85b625-b5d7-41d9-863c-aa56e5f77a9c.png)

From version 1.5.2: You cannot delete selected label (I tried to click anywhere around annotation, use Delete button etc, nothing works for me)
![Zrzut ekranu 2021-11-22 o 13 28 32](https://user-images.githubusercontent.com/62895686/142864699-2533b785-9391-4ce4-a479-59f56cdcf202.png)

*   Operating System: macOS-10.16-x86_64-i386-64bit 
*   Python Version Used: 3.9.6
*   When you install doccano: 22.11.2021 (but also used it in October, and it worked differently back then)
*   How did you install doccano (Heroku button etc): Docker and pip installations (few versions using pip)

Please let me know if you've already seen this issue and if there is anything we can do about it? 

PS. How can I get specific version (1.4.2 for me) on Docker?

**Comment by Hironsan** [comment]
Did you try right click?

**Comment by Joanna-Wojdylo** [comment]
OMG, yup, I tried it, and I was only getting the usual browser options like copy/print etc. BUT now I clicked very carefully  only on the label name and in fact it disappeared, which is great, BUT also this is soooo unintuitive and breaks UI/UX/user-friendly rules.

If it were up to me, I would think about improving/fixing it because it is not intuitive and more people will be as confused as I am today. Also, if there is any information for users about the changes or new design, it is very difficult to find.

Anyway, thank you for such a quick reaction! :) I appreciate it!
